src/composites/FeaturesContainer/Overview/OverviewCollector.ts

```diff
@@ -0,0 +1,103 @@
+import { overviewData } from './index.ts';
+import { State } from "../../../state.ts";
+import { useAtomValue } from "jotai";
+
+//expected DSStats for compact-output: 0-0-0-0-6 / 1-1-0-0-34 / 4-0-1-0-161
+
+export function OverviewDataCollector():overviewData {
+    const dataset = useAtomValue(State.dataset);
+
+    const overviewData: overviewData = {
+        qualityAspectsCount: [0, 0, 0, 0, 0],
+        qualityFactorsCount: [0, 0, 0, 0, 0],
+        measuresCount: [0, 0, 0, 0, 0],
+        qualityAspectNodes: [],
+        qualityFactorNodes: [],
+        measureNodes: [],
+    };
+
+    if(dataset) {
+        for (const measureName in dataset.measures) {
+            if (Object.prototype.hasOwnProperty.call(dataset.measures, measureName)) {
+                const measure = dataset.measures[measureName];
+                
+                overviewData.measureNodes.push({
+                    name: measureName,
+                    value: measure.value,
+                    description: measure.description,
+                    eval_strategy: measure.eval_strategy,
+                    normalizer: measure.normalizer,
+                    utility_function: extractUtilityFunctionName(measure.utility_function),
+                });
+
+                if (measure.value < 0.2) {overviewData.measuresCount[0]++;}
+                else if (measure.value >= 0.2 && measure.value < 0.4) {overviewData.measuresCount[1]++;}
+                else if (measure.value >= 0.4 && measure.value < 0.6) {overviewData.measuresCount[2]++;}
+                else if (measure.value >= 0.6 && measure.value < 0.8) {overviewData.measuresCount[3]++;}
+                else if (measure.value >= 0.8 && measure.value <= 1.0) {overviewData.measuresCount[4]++;}
+            }
+        }
+
+        for (const factorGroupName in dataset.factors) {
+            if (Object.prototype.hasOwnProperty.call(dataset.factors, factorGroupName)) {
```

Comment:
Do we need to check this? At this point the data has been validated and it should contain this key, especially since you're extracting `factorGroupName` from `dataset.factors`

Reply:
No longer checking this part.